### PR TITLE
Include some warnings for Windows

### PR DESCRIPTION
**Pull request by nilsnolde**
One question though:

can you easily build the bindings on mac os? I wonder why they're disabled on CI (and if we can activate them there)..

### CODE REVIEW

**Comment by nilsnolde** [comment]
did you ever try the python bindings on mac @PureW ?

**Comment by PureW** [comment]
> did you ever try the python bindings on mac @PureW ?

I did not unfortunately. I was able to quickly change my mac to a linux machine :)

**Comment by kevinkreiser** [comment]
@nilsnolde iirc the python bindings being disabled in mac was because of a problem with boost python on that platform. now that we no longer use boost python, they might work, ive not tried!

File: README.md [hunk 1]
```diff
@@ -38,6 +38,22 @@ There are several key features that we hope can differentiate the Valhalla proje
 - A plugin based narrative and manoeuvre generation architecture. Should allow for generation that is customized either to the administrative area or to the target locale.
 - Multi-modal and time-based routes. Should allow for mixing auto, pedestrian, bike and public transportation in the same route or setting a time by which one must arrive at a location.
 
+## Platform Compatibility
+
+Valhalla is fully functional on many Linux and Mac OS distributions.
+
+Support for Windows is not yet fully exploited. Building the Valhalla library works flawlessly, as well as the following application modules:
```
Comment (kevinkreiser) [inline README.md:45]:
```suggestion
Support for all functionality in Windows is not yet fully supported. Building the Valhalla library works flawlessly, as well as the following application modules:
```

**Pull request merged**